crates/kobold/js/util.js

```diff
@@ -7,12 +7,7 @@ export function __kobold_before(n,i) { n.before(i); }
 export function __kobold_unmount(n) { n.remove(); }
 export function __kobold_replace(o,n) { o.replaceWith(n); }
 export function __kobold_empty_node() { return document.createTextNode(""); }
-export function __kobold_fragment()
-{
-	let f = document.createDocumentFragment();
-	f.append("", "");
-	return f;
-};
+export function __kobold_fragment() { return document.createDocumentFragment().append("", ""); };
```

Comment:
`append` returns `undefined` so this change bricks this function

Reply:
thanks for the feedback! i've just been reading through chapter 4 of the rust book so i'm extra conscious of trying to remove any variables if their not necessary. i'll have a more detailed look at the code now.